trie/node.go

```diff
@@ -127,8 +127,8 @@ func (b *branch) Encode() ([]byte, error) {
 
 	for _, child := range b.children {
 		if child != nil {
-			encChild, err := Encode(child)
-			if err != nil {
+			encChild, e := Encode(child)
```

Comment:
idk why this is changed to e? not a big deal tho

Reply:
The linter complained that `err` was overshadowed by the previous declaration